suites/apis/dbgapTest.js

```diff
@@ -250,6 +285,73 @@ Scenario('dbGaP Sync: created signed urls (from s3 and gs) to download, try crea
       + 'project phs000178, even though they have read access.');
 }).retry(1);
 
+Scenario('dbGaP Sync: Cascading Auth - create signed urls from s3 and gs to download, @dbgapSyncing @reqGoogle',
+    async ({I, fence, users}) => {
+        console.log('Use mainAcct to create s3 signed URL for file phs001194');
+        const signedUrls3Phs001194Res = await fence.do.createSignedUrl(
+            indexed_files.parentPhs001194File.did, ['protocol=s3'],
+            users.mainAcct.accessTokenHeader,
+        );
+
+        console.log('Use mainAcct to create gs signed URL for file phs001194');
+        const signedUrlgsPhs001194Res = await fence.do.createSignedUrl(
+            indexed_files.parentPhs001194File.did, ['protocol=gs'],
+            users.mainAcct.accessTokenHeader,
+        );
+        const signedUrls3phs001194Res = await fence.do.createSignedUrl(
+            indexed_files.parentPhs001194File.did, ['protocol=s3'],
+            users.mainAcct.accessTokenHeader,
+        );
```

Comment:
Why are we creating presigned url for 1194 twice with s3 protocol?

Reply:
The second s3 creation has been removed.